modules/formulize/class/elements.php

```diff
@@ -109,7 +109,7 @@ function getDataTypeInformation() {
 			$defaultTypeSize = '';
 		}
 		//define array and return type and size
-		return array("dataType" => $defaultType, "dataTypeSize" => $defaultTypeSize);
+		return array("dataType" => $defaultType, "dataTypeSize" => $defaultTypeSize, "dataTypeString" => $defaultTypeComplete);
```

Comment:
What are your thoughts on this. Maybe `dataTypeCompleteString`?

Reply:
Yes, I think that would be better. There's only two hard things in computer science...